tests/unit/parsers/test.manifestjson.js

```diff
@@ -277,6 +277,35 @@ describe('ManifestJSONParser', () => {
     expect(manifestJSONParser.isValid).toEqual(true);
   });
 
+  it('should accept applications.gecko AND browser_specific_settings.gecko_android', () => {
+    const addonLinter = new Linter({ _: ['bar'] });
+    const json = validManifestJSON({
+      applications: {
+        gecko: {
+          strict_max_version: '100.0',
+        },
+      },
+      browser_specific_settings: {
+        gecko_android: {},
+      },
+    });
+
+    const manifestJSONParser = new ManifestJSONParser(
+      json,
+      addonLinter.collector
+    );
```

Comment:
eslint is complaining that manifestJSONParser isn't used, we should fix this nit before landing this PR.